step03/models.py

```diff
@@ -0,0 +1,49 @@
+from sqlalchemy import Column, String, Integer, ForeignKey, Boolean
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import relationship
+
+Base = declarative_base()
```

Comment:
I would prefer that you use flask_migrate to create your models so you can evolve the schema as needed without destroying data